<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="pagebuttons">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="frame_pages">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_1">
        <widget class="QLabel" name="welcomeLabel">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>100</y>
           <width>169</width>
           <height>78</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome to HelloWorld!</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>190</y>
           <width>321</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="enterNamePrompt">
            <property name="text">
             <string>Enter your name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameFieldLE"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="DOBPrompt">
            <property name="text">
             <string>Date of Birth</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="DOBFieldLE"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="enterBtn">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>280</y>
           <width>151</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>70</y>
           <width>131</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="genderLabel">
            <property name="text">
             <string>Gender:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="genderBtn_Male">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="genderBtn_Female">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>70</y>
           <width>166</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="occLabel">
            <property name="text">
             <string>Occupation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Twitch Streamer</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Software Engineer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Chef</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>Police</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Secret Agent</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>151</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="weightLabel">
            <property name="text">
             <string>Weight (kg)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="heightLabel">
            <property name="text">
             <string>Height (m)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="weightSpinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="heightSpinBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>300</y>
           <width>301</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="readFromRegBtn">
            <property name="text">
             <string>Read from Windows Registry</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="writeToRegBtn">
            <property name="text">
             <string>Write to Windows Registry</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>701</width>
           <height>291</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Preset 1</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>40</y>
             <width>361</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Date of Birth:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Occupation:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Gender:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Weight:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit_8"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Preset 2</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>40</y>
             <width>361</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_6">
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_15"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Date of Birth:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_16"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Occupation:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_17"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Gender:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Weight:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_18"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_19"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit_20"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Preset 3</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>40</y>
             <width>361</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_9"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Date of Birth:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_10"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Occupation:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_11"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Gender:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Weight:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_12"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_13"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit_14"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>310</y>
           <width>271</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="loadPreBtn">
            <property name="text">
             <string>Load Preset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePreBtn">
            <property name="text">
             <string>Save Preset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
